### PR TITLE
Base 404

### PR DESCRIPTION
**Pull request by kieftrav**
_Please request review from `@zooniverse/frontend` team or an individual member of that team._ 

## Linked Issue and/or Talk Post
- Subset of the [Custom 404 PR](https://github.com/zooniverse/front-end-monorepo/pull/4894)

## Describe your changes
Subset of the Figma design for the 404 page. 

What Changed:
- Added Header and Footer
- Rotating background image
- Link to Home Page and Projects Page

What's Missing:
- Link back to the Project's page (related to Next.js processing of 404 page)
- Internationalization of 404 page content (TBD if possible with follow up PR)

Links to See Variability:
- [Random 404](https://local.zooniverse.org:3000/projects/test404)
- [Gravity Spy English](https://local.zooniverse.org:3000/projects/zooninverse/gravity-spy)
- [Gravity Spy Spanish](https://local.zooniverse.org:3000/projects/es/zooninverse/gravity-spy)

## General
- [ ] Tests are passing locally and on Github
- [ ] Documentation is up to date and changelog has been updated if appropriate
- [ ] You can `yarn panic && yarn bootstrap` or `docker-compose up --build` and FEM works as expected
- [ ] FEM works in all major desktop browsers: Firefox, Chrome, Edge, Safari (Use Browserstack account as needed)
- [ ] FEM works in a mobile browser

## General UX
Example Staging Project: [i-fancy-cats](https://local.zooniverse.org:3000/projects/brooke/i-fancy-cats)
- [ ] All pages of a FEM project load: Home Page, Classify Page, and About Pages
- [ ] Can submit a classification
- [ ] Can sign-in and sign-out
- [ ] The component is accessible
  - Can be used with a screen reader [BBC guide to VoiceOver](https://bbc.github.io/accessibility-news-and-you/assistive-technology/testing-steps/voiceover-mac.html)
  - Can be used from the keyboard [WebAIM guide to keyboard testing](https://webaim.org/techniques/keyboard/#testing)
  - It is passing accessibility checks in the storybook

### CODE REVIEW

**Comment by kieftrav** [comment]
@seanmiller26 - take a look at the Base 404 page here:

- [Random 404](https://fe-project-branch.preview.zooniverse.org/projects/test404)
- [Gravity Spy English](https://fe-project-branch.preview.zooniverse.org/projects/zooninverse/gravity-spy)
- [Gravity Spy Spanish](https://fe-project-branch.preview.zooniverse.org/projects/es/zooninverse/gravity-spy)

**Comment by seanmiller26** [comment]
On initial page load / refresh I can see other images flicker for a second. Is this something that can be remedied?


https://github.com/zooniverse/front-end-monorepo/assets/89043489/f1a50af2-5e60-49f3-9ace-030285f29b9d

**Comment by eatyourgreens** [comment]
I’ve made a small change which should make sure that the random image is only generated once.

File: packages/app-project/pages/404.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+  z-index: 1;
+`
+
+const Overlay = styled(Box)`
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  opacity: 0.5;
+  z-index: 2;
```
Comment (eatyourgreens) [inline packages/app-project/pages/404.js:20]: We can remove this layer entirely, and replace it with a [CSS filter](https://developer.mozilla.org/en-US/docs/Web/CSS/filter) on the background image. That would make the page structure a lot simpler.

**Review by eatyourgreens — APPROVED** [review]
LGTM. Maybe replace the semi-transparent overlay with an opacity filter on the background image?

**Comment by seanmiller26** [comment]
@eatyourgreens The intention is to have a random image appear each time. It looks like with your 'fix' the same image loads. @kieftrav can you confirm this?

**Comment by eatyourgreens** [comment]
My latest commit should have fixed that.

**Comment by eatyourgreens** [comment]
There is an ugly Flash Of Unstyled Content when the page first loads.


https://github.com/zooniverse/front-end-monorepo/assets/59547/3679f5d5-9929-4174-8ade-2f0b0dae0511

**Comment by seanmiller26** [comment]
The randomized image looks fine now. Agreed on the unstyled content. That needs some attention

**Pull request merged**